src/mds/MDSDaemon.cc

```diff
@@ -373,14 +373,14 @@ void MDSDaemon::set_up_admin_socket()
 				     "Enumerate connected CephFS clients");
   ceph_assert(r == 0);
   r = admin_socket->register_command("session config "
-				     "name=client_id,type=CephInt,req=true "
```

Comment:
Seems to me the help output for asok commands should be improved. This is just circumventing the underlying problem.